scripts/encode_data.py

```diff
@@ -93,7 +122,7 @@ def _load_files(self) -> None:
                 name, extension = os.path.splitext(full_path) 
 
                 match extension:
-                    case ".pdf" | ".rst":
+                    case ".pdf" | ".rst" | ".md":
```

Comment:
If you add markdown here you need to add a parser for markdown in the process function